pkg/packages/imports.go

```diff
@@ -137,7 +137,7 @@ func parseTypeDec(s string) (string, string) {
 func isBuiltIn(s string) bool {
 	s = strings.NewReplacer("*", "", "[]", "").Replace(s)
 	switch s {
-	case "bool", "string", "int", "int64", "map[string]string":
+	case "range", "bool", "string", "int", "int64", "map[string]string":
```

Comment:
`isBuiltIn` function is created to filter the native types but `range` is not one. So, I think having another function for language keywords could make more sense. Then we can use that function to decide whether to prepend `_` instead of prepending to all.